float64 x
float64 y
float64 theta
float64 width
float64 height
bool available
int32 count